File: orientation-post-baccalauriat-BackEnd/src/main/java/com/biorbac/dto/InstitutionDto.java
package com.biorbac.dto;

import com.biorbac.enums.InstitutionType;
import lombok.Getter;
import lombok.Setter;

@Getter
@Setter
public class InstitutionDto {

    private Long institutionId;
    private String institutionName;
    private String description;
    private String address;
    private String website;
    private String institutionLogo;
    private Double averageRating;
    private InstitutionType institutionType;

}

File: orientation-post-baccalauriat-FrontEnd/src/app/core/Dto/institution-dto.ts
import {InstitutionType} from "../enums/institution-type";

export interface InstitutionDto {
  institutionId: number;
  institutionName: string;
  description: string;
  address: string;
  website: string;
  institutionLogo: string;
  averageRating: number;
  institutionType: InstitutionType;
}

